styles.css

```
@@ -4,279 +4,476 @@ body {
+  border: 1px solid #d0d9d4;
+  border-radius: 15px;
+  margin-bottom: 150px;
+  height: min-content;
+}
+
+.placehold {
+  height: 252px;
+}
+
+.flex-container h3 {
+  text-align: center;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: #3a4a42;
+}
+
+.flex-container ul {
+  display: flex;
+  align-items: center;
+  padding: 0;
+  justify-content: center;
+}
+
+.flex-container li {
+  flex: none;
+  list-style-type: none;
+  padding: 8px 12px;
+  margin: 0 8px;
+  border-radius: 4px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  color: #3a4a42;
+  background: #ebf0ee;
+}
+
+.aboutme-text {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #42526e;
+  margin-top: 40px;
+}
+
+.flex-container2 {
+  padding: 50px;
+  gap: 100px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.nest-flex {
+  background: #ebf0ee;
+  border: 1px solid #c1c7d0;
+  border-radius: 8px;
+  text-align: center;
+  list-style: none;
+  height: 352px;
+  width: 327px;
+  display: flex;
+  flex-direction: column;
+}
+
+.nest-flex ul {
+  list-style: none;
+  padding-left: 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.nest-flex img {
+  order: -1;
+  padding-top: 20px;
+}
+
+.nest-flex h3 {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: #344563;
+}
+
+.nest-flex li {
+  flex: none;
+  list-style-type: none;
+  padding: 8px 12px;
+  margin: 8px;
+  border-radius: 8px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  background: #fff;
+  color: #36b37e;
+}
+
+.background_aboutme {
+  background-image: url(./images/aboutme1.svg), url(./images/aboutme2.svg);
+  background-position: left bottom, right top;
+  background-repeat: no-repeat, no-repeat;
+}
+
+footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url(./images/contact-background.svg);
+  background-repeat: no-repeat;
+  background-position: right 90%;
+}
+
+.contact-heading {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+footer form {
+  margin-top: 58px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 90%;
+}
+
+footer input {
+  margin: 5%;
+  background: #fff;
+  border: 1px solid #d0d9d4;
+  box-sizing: border-box;
+  border-radius: 4px;
+  padding: 15px 16px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: #6b778c;
+  width: 100%;
+}
+
+footer textarea {
+  padding: 12px;
+  border-radius: 4px;
+  border: 1px solid #60c095;
+  background: #fff;
+  margin: 5%;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: #091e42;
+  width: 92.5%;
```

Here 👇 you can do this to fix the aforementioned issue.
```suggestion
  width: 100%;
  box-sizing: border-box;
```